--- CMakeLists.txt.build	2011-05-05 11:49:28.000000000 +0200
+++ CMakeLists.txt	2011-05-05 11:49:47.000000000 +0200
@@ -44,7 +44,7 @@
 # Add some gcc-specific command lines.
 IF(CMAKE_COMPILER_IS_GNUCC)
     # Always build with debug symbols...you can strip it later.
-    ADD_DEFINITIONS(-g -pipe -Werror -fsigned-char)
+    ADD_DEFINITIONS(-g -pipe -fsigned-char)
 
     # Stupid BeOS generates warnings in the system headers.
     IF(NOT BEOS)
